examples/eco/05-offshore-h2/input/plant/orbit-config-osw_18MW.yaml

```diff
@@ -214,37 +214,40 @@ policy_parameters: # these should be adjusted for inflation prior to application
 plant_design:
   scenario0:
     electrolyzer_location: "platform" # can be one of ["onshore", "turbine", "platform"]
-    transportation: "pipeline" # can be one of ["hvdc", "pipeline", "none"]
+    transportation: "pipeline" # can be one of ["hvdc", "pipeline", "none", hvdc+pipeline]
     h2_storage_location: "platform" # can be one of ["onshore", "turbine", "platform"]
   scenario1:
     electrolyzer_location: "onshore" # can be one of ["onshore", "turbine", "platform"]
-    transportation: "hvdc" # can be one of ["hvdc", "pipeline", "none"]
+    transportation: "hvdc" # can be one of ["hvdc", "pipeline", "none", hvdc+pipeline]
```

Comment:
If `electrolyzer_location: "onshore"` then `hvdc+pipeline` should not be an option

Reply:
We needed that combination for a project that requested it. The electrolyzer was onshore and the h2 storage was offshore.